lib/autokey/scripting/engine.py

```diff
@@ -483,44 +470,68 @@ def remove_all_temporary(self, folder=None,
         self.configManager.remove_all_temporary(folder,
                 in_temp_parent)
 
+    def validateArguments(self, folder, name, contents,
```

Comment:
Please move `validateArguments` out of the class. It can be static method, so it is safe to move out. That way, it will be inaccessible to user scripts.